admiral/pkg/clusters/registry.go

```diff
@@ -83,6 +88,24 @@ func InitAdmiral(ctx context.Context, params common.AdmiralParams) (*RemoteRegis
 	return &w, nil
 }
 
+func pauseForAdmiralToInitializeState(){
+	// Sleep until Admiral determines state. This is done to make sure events are not skipped during startup while determining READ-WRITE state
+	start := time.Now()
+	log.Info("Pausing thread to let Admiral determine it's READ-WRITE state. This is to let Admiral determine it's state during startup")
+	for {
+		if CurrentAdmiralState.IsStateInitialized {
+			log.Infof("Time taken for Admiral to complete state initialization =%v ms", time.Since(start).Milliseconds())
+			break
+		}
+		if time.Since(start).Milliseconds() > 60000 {
+			log.Error("Admiral not initialized after 60 seconds. Exiting now!!")
+			os.Exit(-1)
+		}
```

Comment:
Add a debug log statement to say Admiral is waiting for state check before it boots up

Reply:
I have a info log -> https://github.com/asushanthk/admiral/blob/dr-support-bk/admiral/pkg/clusters/registry.go#L94
Do you want me to change line no 94 ?